v1.0/openapi3/parameters/SubId.yaml

```diff
@@ -3,4 +3,6 @@ in: path
 required: true
 schema:
   type: string
-description: 'A sub-identifier of the party identifier, or a sub-type of the party identifier''s type. For example, `PASSPORT`, `DRIVING_LICENSE`.'
+description: |
+  A sub-identifier of the party identifier, or a sub-type of the party
+  identifier''s type. For example, `PASSPORT`, `DRIVING_LICENSE`.
```

Comment:
Just checking, do we still need double apostrophes here? Would it result in one or two chars since we now use a multiline description?

Reply:
Good spot.